packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -11,10 +11,824 @@ import 'picker.dart';
 /// Default aesthetic values obtained by comparing with iOS pickers.
```

Comment:
All public classes in this file should have a `See also:` section in their class doc that says something like consider also using CupertinoPicker for a content agnostic spinner UI